docs/examples/opentelemetry-example-app/setup.py

```diff
@@ -20,7 +20,7 @@
     author="OpenTelemetry Authors",
     author_email="cncf-opentelemetry-contributors@lists.cncf.io",
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```

Comment:
Is this really OK? Is the spec even in beta?

Reply:
All this time we have been talking about releasing the beta version, v0.5.0 is marked as beta and next week we are going to have another beta release. Am I missing something?

Reply:
I haven't been following the discussion really. I just had the feeling that the spec is not really stable yet, with discussions like https://github.com/open-telemetry/opentelemetry-specification/issues/514 or https://github.com/open-telemetry/oteps/issues/73

Reply:
I agree the spec is a moving target, but we're only targeting the [0.3 spec release](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/0.3) with our 0.5.x releases. Different companies and projects might reasonably disagree on what constitutes a "beta", but I think it does make sense to use the "beta" classifier here since that's what we're calling the release elsewhere.